composer.json

```diff
@@ -21,19 +21,22 @@
         "psr/log": "^1.1",
         "psr/event-dispatcher": "^1.0",
         "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/html": "^3.0@dev",
         "yiisoft/var-dumper": "^3.0@dev",
         "yiisoft/files": "^3.0@dev"
     },
     "require-dev": {
+        "friendsofphp/php-cs-fixer": "^2.2",
         "hiqdev/composer-config-plugin": "^1.0@dev",
+        "phpunit/phpunit": "^8.0",
+        "yiisoft/aliases": "^3.0@dev",
+        "yiisoft/cache": "^3.0@dev",
         "yiisoft/di": "^3.0@dev",
+        "yiisoft/event-dispatcher": "^3.0@dev",
         "yiisoft/log": "^3.0@dev",
-        "yiisoft/cache": "^3.0@dev",
-        "yiisoft/yii-core": "^3.0@dev",
         "yiisoft/strings": "^3.0@dev",
-        "yiisoft/event-dispatcher": "^3.0@dev",
-        "phpunit/phpunit": "^7.3",
-        "friendsofphp/php-cs-fixer": "^2.2"
+        "yiisoft/yii-core": "^3.0@dev"
```

Comment:
This dependency should be removed.

Reply:
I use the UrlHelper to check the relative URL, we should create a static YII helper for small checks, such as isUrlRelative, isUrlAboslute, getVersionYii, simple things that we need to access from many sites and not repeat the code.

Reply:
We can't create URL helper. Request is not globally available.

Yii version could be obtained via https://github.com/yiisoft/yii-web/blob/master/src/Info.php

Reply:
I know that we do not have the UrlHelper in a global way but we should not remove the check, on the other hand I think we must create then Info.php does not make sense to add a dependency to return the package version.

Reply:
For isUrlRelative, isUrlAboslute let's copy these into view package directly for now.